docs/guide-authentication.md

```diff
@@ -58,17 +58,90 @@ Once enabled the sign in dialog will display additional sign in options:
 OpenID Connect clients can be configured by adding entities to the 'OIDC client' entity type (e.g. using the data explorer, importer or REST API).
-The required information is provided by the identity provider. MOLGENIS requires at least the 'sub' and 'email' claims and ideally the
-'given_name' and 'family_name' claims as well. These claimes are requested by specifying the scopes 'openid,email,profile'.  
 
-#### Activating OpenID Connect clients
+#### Registration ID
+This is the registration ID that ends up in the redirect URI on the MOLGENIS server.
+
+#### Client ID and secret
+You need to fill in the client ID and client secret that you got from the ID
+provider.
+
+#### Client name
+How the end user sees the identity provider.
+This becomes the label on the button on the login page.
+
+#### Scopes and claims
+MOLGENIS requires at least the `sub` and `email` claims and ideally the `given_name`
+and `family_name` claims as well.
+These claims are requested by specifying the scopes `openid,email,profile`.
+
+#### Endpoints
+The identity provider specifies a set of endpoints where the user and MOLGENIS
+can interact with the provider. These are provided by the ID provider but may
+be nontrivial to find. Here are suggested values for a couple of common OpenID providers:
+
+##### Google
+
+|  | value |
+| --- | --- |
+| Authorization URI  | `https://accounts.google.com/o/oauth2/v2/auth` |
+| Token URI          | `https://www.googleapis.com/oauth2/v4/token` |
+| JWKS URI           | `https://www.googleapis.com/oauth2/v3/certs` |
+| User info URI      | `https://www.googleapis.com/oauth2/v3/userinfo` |
+| Username attribute | `sub` |
+
+(See: `https://accounts.google.com/.well-known/openid-configuration`)
+
+##### Auth0
+
+|  | value |
+| --- | --- |
+| Authorization URI  | `https://<realm>.auth0.com/authorize` |
+| Token URI          | `https://<realm>.auth0.com/oauth/token` |
+| JWKS URI           | `https://<realm>.auth0.com/.well-known/jwks.json` |
+| User info URI      | `https://<realm>.auth0.com/userinfo` |
+| Username attribute | `sub` |
+
+(See `https://<realm>.auth0.com/.well-known/openid-configuration`)
+
+##### Keycloak
+
+|  | value |
+| --- | --- |
+| Authorization URI  | `https://<server>/auth/realms/<realm>/protocol/openid-connect/auth` |
+| Token URI          | `https://<server>/auth/realms/<realm>/protocol/openid-connect/token` |
+| JWKS URI           | `https://<server>/auth/realms/<realm/protocol/openid-connect/certs` |
+| User info URI      | `https://<server>/auth/realms/<realm>/protocol/openid-connect/userinfo` |
+| Username attribute | `sub` |
+
+(See `https://<server>/auth/realms/<realm>/.well-known/openid-configuration`)
+
+### Activating OpenID Connect clients
 Configured OpenID Connect clients can be selected in the 'Authentication settings' part of the 'Settings manager' plugin. Activating clients
 requires 'Allow users to sign up' to be set to 'Yes' and 'Sign up moderation' to be set to 'No'.
```

Comment:
And in 'Authentication server' one or more OpenID Connect clients needs to be added. Only the clients that are created in entity 'OIDC client' can be selected.
![autentication](https://user-images.githubusercontent.com/40240913/70709730-e1ced480-1cdd-11ea-855b-6016630b152c.png)